frontend/src/App.js

```diff
@@ -14,12 +15,15 @@ function App() {
             <BrowserRouter>
                 <Routes>
                     <Route path="/" element={<Layout />}>
+
                         {/* public routes */}
                         <Route element={<PersistLogin />}>
                             <Route path="/signup" element={<Signup />} />
                             <Route path="/login" element={<Login />} />
+                            <Route path="/question" element={<Question />} />

```

Comment:
The question page does not require log in right now?

Reply:
It's just for testing purposes, will remove